lib/vmpooler/api.rb

```diff
@@ -189,23 +188,17 @@ def hostname_shorten hostname
               result['empty'] ||= []
               result['empty'].push( pool['name'] )
             end
-
-            $redis.smembers( 'vmpooler__ready__'+pool['name'] ).each do |vm|
-              clone_time = $redis.hget( 'vmpooler__vm__'+vm, 'clone_time' )
-              clone_times.push( clone_time ) if clone_time.to_f > 0
-            end
           end
 
           if ( result['empty'] )
             result['status'] = 0
           end
 
-          if ( clone_times.any? )
-            result['clone_average'] = clone_times.map( &:to_f ).reduce( :+ ) / clone_times.size
-          end
-
           result['capacity_perecent'] = ( result['capacity_current'].to_f / result['capacity_total'].to_f ) * 100.0
 
+          result['clone_total'] = $redis.hlen('vmpooler__clone__'+Date.today.to_s)
```

Comment:
This is total for today, not for 24 hours. I do not see a piece that calculates `now - 24 hours`.